[2024-05-28T02:13:31.176+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: franko_lookup_ingest.ti_load_target_data manual__2024-05-27T21:49:08.726117+00:00 [queued]>
[2024-05-28T02:13:31.194+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: franko_lookup_ingest.ti_load_target_data manual__2024-05-27T21:49:08.726117+00:00 [queued]>
[2024-05-28T02:13:31.194+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-05-28T02:13:31.195+0000] {taskinstance.py:1284} INFO - Starting attempt 16 of 16
[2024-05-28T02:13:31.195+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-05-28T02:13:31.223+0000] {taskinstance.py:1304} INFO - Executing <Task(DockerOperator): ti_load_target_data> on 2024-05-27 21:49:08.726117+00:00
[2024-05-28T02:13:31.231+0000] {standard_task_runner.py:55} INFO - Started process 6658 to run task
[2024-05-28T02:13:31.235+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'franko_lookup_ingest', 'ti_load_target_data', 'manual__2024-05-27T21:49:08.726117+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/challenge_dag.py', '--cfg-path', '/tmp/tmp2tvp3n8x']
[2024-05-28T02:13:31.235+0000] {standard_task_runner.py:83} INFO - Job 192: Subtask ti_load_target_data
[2024-05-28T02:13:31.315+0000] {task_command.py:389} INFO - Running <TaskInstance: franko_lookup_ingest.ti_load_target_data manual__2024-05-27T21:49:08.726117+00:00 [running]> on host a64b6410e10e
[2024-05-28T02:13:31.420+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=franko_lookup_ingest
AIRFLOW_CTX_TASK_ID=ti_load_target_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-27T21:49:08.726117+00:00
AIRFLOW_CTX_TRY_NUMBER=16
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-27T21:49:08.726117+00:00
[2024-05-28T02:13:31.450+0000] {docker.py:284} INFO - Starting docker container from image my_dbt
[2024-05-28T02:13:34.382+0000] {docker.py:354} INFO - [0m02:13:34  Running with dbt=1.5.11
[2024-05-28T02:13:34.494+0000] {docker.py:354} INFO - [0m02:13:34  Registered adapter: postgres=1.5.11
[2024-05-28T02:13:34.607+0000] {docker.py:354} INFO - [0m02:13:34  Found 6 models, 4 tests, 0 snapshots, 0 analyses, 422 macros, 0 operations, 1 seed file, 1 source, 0 exposures, 0 metrics, 0 groups
[2024-05-28T02:13:34.609+0000] {docker.py:354} INFO - [0m02:13:34
[2024-05-28T02:13:34.679+0000] {docker.py:354} INFO - [0m02:13:34  Concurrency: 4 threads (target='dev')
[2024-05-28T02:13:34.679+0000] {docker.py:354} INFO - [0m02:13:34
[2024-05-28T02:13:34.682+0000] {docker.py:354} INFO - [0m02:13:34  1 of 2 START sql table model dev.golden_data ................................... [RUN]
[2024-05-28T02:13:39.734+0000] {docker.py:354} INFO - [0m02:13:39  1 of 2 OK created sql table model dev.golden_data .............................. [[32mSELECT 323469[0m in 5.05s]
[2024-05-28T02:13:39.736+0000] {docker.py:354} INFO - [0m02:13:39  2 of 2 START sql table model dev.golden_mart_data .............................. [RUN]
[2024-05-28T02:13:39.796+0000] {docker.py:354} INFO - [0m02:13:39  2 of 2 OK created sql table model dev.golden_mart_data ......................... [[32mSELECT 1[0m in 0.06s]
[2024-05-28T02:13:39.905+0000] {docker.py:354} INFO - [0m02:13:39
[2024-05-28T02:13:39.906+0000] {docker.py:354} INFO - [0m02:13:39  Finished running 2 table models in 0 hours 0 minutes and 5.30 seconds (5.30s).
[2024-05-28T02:13:39.919+0000] {docker.py:354} INFO - [0m02:13:39
[2024-05-28T02:13:39.919+0000] {docker.py:354} INFO - [0m02:13:39  [32mCompleted successfully[0m
[2024-05-28T02:13:39.920+0000] {docker.py:354} INFO - [0m02:13:39
[2024-05-28T02:13:39.920+0000] {docker.py:354} INFO - [0m02:13:39  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-05-28T02:13:40.740+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=franko_lookup_ingest, task_id=ti_load_target_data, execution_date=20240527T214908, start_date=20240528T021331, end_date=20240528T021340
[2024-05-28T02:13:40.772+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-05-28T02:13:40.793+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
